lib/opensusebasetest.pm

```diff
@@ -213,9 +213,11 @@ sub wait_boot {
 
     # Reset the consoles after the reboot: there is no user logged in anywhere
     reset_consoles;
-    assert_screen "bootloader", $bootloader_time;
+    if (get_var("OFW")) {
+        assert_screen 'bootloader', $bootloader_time;
```

Comment:
why you think is appropriate to have two tags "bootloader" and "bootloader-ofw" while they have identical needles ?
Test you mention is failing because correct needles with "bootloader" tag is having ENV-VERSION-12-SP3 tag so the fix from me will be to remove this tag